src/main/webapp/createGame.html

```diff
@@ -44,18 +44,20 @@
       <!-- Div for the hints -->
       <div id='hints'>
         <div id='stage1Hints'>
-          <input type='text' id='started-postion' placeholder='Starter Postion (click on map to get corrdinates)'>
+          <input type='text' id='key' placeholder='Stage Key'>
+          <input type='text' id='starter-position' placeholder='Starter Position (click on map to get corrdinates)'>
           <input type='text' placeholder='Starter Hint' id='starter-hint' required>
-          <input type='text' id='hint1-postion' placeholder='Hint 1 Postion (click on map to get corrdinates)'>
+          <input type='text' id='hint1-position' placeholder='Hint 1 Position (click on map to get corrdinates)'>
           <input type='text' placeholder='Hint 1' id='hint1'>
         </div> 
       </div>
       <input type='button' value='+ add hint' id='addHint' onclick='addNewHint()'> 
 
       <br/>
-      <input type='button' value='Finish'>
+      <input type='button' value='Finish' onclick='getDataFromForm()'>
```

Comment:
Maybe rename to 'getDataFromGameCreationForm' because we're going to have multiple forms in the future.

Reply:
Makes sense, will do!